features/step_definitions/metric_configuration_steps.rb

```diff
@@ -41,6 +41,7 @@
   expect(page).to have_content(@metric_configuration.metric.name)
   expect(page).to have_content(@metric_configuration.metric.code)
   expect(page).to have_content(@metric_configuration.weight)
+  expect(page).to have_content("Percentage of use")
```

Comment:
Is this expectation necessary? It is not redundant with the one at  features/metric_configuration/create.feature on line 102?